docs/architecture/raft.rst

```diff
@@ -237,34 +237,18 @@ In summary, Raft makes schema changes safe, but it requires that a quorum of nod
 
 .. _raft-topology-changes:
 
-.. only:: opensource
-
-    Consistent Topology with Raft :label-caution:`Experimental`
-    -----------------------------------------------------------------
-
```

Comment:
Was is removed from the code too?

Reply:
It's work in progress: https://github.com/scylladb/scylladb/issues/17802